lsp-clients.el

```diff
@@ -609,17 +609,22 @@ responsiveness at the cost of possible stability issues."
 
 
 
-;; Nim
-(defgroup lsp-nim nil
-  "LSP support for Nim, using nimlsp."
+;; Crystal
+(defgroup lsp-crystal nil
+  "LSP support for Crystal via scry."
   :group 'lsp-mode
-  :link '(url-link "https://github.com/PMunch/nimlsp"))
+  :link '(url-link "https://github.com/crystal-lang-tools/scry"))
+
+(defcustom lsp-clients-crystal-executable '("scry" "--stdio")
+  "Command to start the scry language server."
+  :group 'lsp-crystal
+  :risky t
+  :type 'file)
```

Comment:
I think this is wrong, right?
crystal already has its own server on `lsp-crystal.el`

Reply:
fixed!

PS: It seems github do not notify every conversation...